common/changes/@itwin/imodel-browser-react/rerender-grid-on-action_2024-12-18-16-51.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/imodel-browser-react",
+      "comment": "provide option to refresh the grid after an itwin/imodel action",
+      "type": "minor"
```

Comment:
This is not a minor version bump. It is a breaking change since you are adding a mandatory forceRefresh property. Does that really need to be mandatory?

Reply:
the forceRefresh is optional. If you check the packages/modules/imodel-browser/src/utils/_buildMenuOptions.tsx file, this is the action menu that the client consumes. The rest of the locations are all internal to this package

Reply:
forceRefresh is a mandatory property of ITwinTileProps, which is exported from the package for public use from the index.ts.

Any consumer who is using iTwinTileProps will break once they update.

Reply:
oh you're right. Ok great I'll make it optional in ITwinTile and iModelTile